portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -92,4 +106,18 @@ private String getParameter(HttpServletRequest request, String name, String defa
     }
     return value;
   }
+
+  /**
+   * Inner class for Comments.
```

Comment:
It would be better to have this comment give more of a description of what a 'Comment' is

Reply:
Good advice -- added a bit of a fuller explanation.